### ISSUE TITLE
role and aria-* attributes should be allowed if img has accessible name via aria-label, aria-labelledby

### ISSUE THREAD

**Issue by dd8** [validator/validator#1198]
The following code has an accessible name from the aria-label, 

```
<img src="test.png" aria-label="Has a name, so can have a role" role="img" />
```

but is showing these messages:

```
An img element with no alt attribute must not have a role attribute
An img element with no alt attribute must not have any aria-* attributes other than aria-hidden
```

That doesn't seem to match the spec:

>If no accessible name is provided via other img naming methods (e.g., aria-labelledby, aria-label): No role, and no aria-* attributes except aria-hidden="true".

https://www.w3.org/TR/html-aria/#el-img-no-alt

**Comment by pkra** [comment]
I think this is the same as #1599.

**Comment by scottaohara** [comment]
#1599 was closed as being a duplicate of this issue - but it was the issue i opened to indicate that ARIA in HTML was updated to modify the img element allowances, which should help close this issue.
